unittests/allplatformstests.py

```diff
@@ -1927,6 +1927,33 @@ def test_options_with_choices_changing(self) -> None:
                 self.assertEqual(item['value'], ['b', 'c'])
                 self.assertEqual(item['choices'], ['b', 'c', 'd'])
 
+    def test_options_listed_in_build_options(self) -> None:
+        """Detect when changed options become listed in build options."""
+        testdir = os.path.join(self.unit_test_dir, '110 list build options')
+
+        out = self.init(testdir)
+        for line in out.splitlines():
+            if line.startswith('Message: Build options:'):
+                self.assertIn('-Dauto_features=auto', line)
```

Comment:
Did you mean `assertNotIn` here? This is causing the tests to fail, but if I change it to `assertNotIn`, it passes for me locally.

Reply:
Good catch!